todo_app/templates/index.html

```diff
@@ -6,22 +6,61 @@
     <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
-
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        {% for item in items %}
-          <li>{{item.title}}</li>
-        {% endfor %}
-      </ul>
-    </div>
+  <div class="my-3 p-3 bg-white rounded shadow-sm">
+    <h3 class="border-bottom border-gray pb-2 mb-0">Not Started</h3>
+    {% for item in items %}
+      {% if item.status == "Not Started" %}
+        <div class="media text-muted pt-3">
+          <!--.<svg class="bd-placeholder-img mr-2 rounded" width="32" height="32" xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid slice" focusable="false" role="img" aria-label="Placeholder: 32x32"><title>Placeholder</title><rect width="100%" height="100%" fill="#007bff"></rect><text x="50%" y="50%" fill="#007bff" dy=".3em">32x32</text></svg>-->
+          <div class="media-body pb-3 mb-0 small lh-125 border-bottom border-gray">
+            <div class="d-flex justify-content-between align-items-center w-100">
+              <strong class="text-gray-dark">{{item.title}}</strong>
+              <div class="btn-group btn-group-sm" role="group" aria-label="Basic example">
+                <a type="button" href="/complete/{{item.id}}" class="btn btn-secondary">Complete</a>
+                <a type="button" href="/remove/{{item.id}}" class="btn btn-secondary">Remove</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      {% endif %}
+    {% endfor %}
+  </div>
+  <div class="my-3 p-3 bg-white rounded shadow-sm">
+    <h3 class="border-bottom border-gray pb-2 mb-0">Add Item</h3>
+    <form action="/" method="post">
+      <div class="input-group mb-3">
+        <input type="text" class="form-control" placeholder="Recipient's username" aria-label="Recipient's username" aria-describedby="button-addon2" id="addItem" name="addItem">
```

Comment:
The placeholder here doesn't make sense in this context.

Reply:
I think this must've been left over from copy & pasting from the bootstrap examples...